toolkits/nature/packages/nature-facet/scss/50-components/facet.scss

```diff
@@ -0,0 +1,171 @@
+.c-facet {
+	background-color: #f3f3f3;
+	padding: spacing(8);
+	position: relative;
+	font-size: 1.6rem;
+	color: black;
+
+	@include media-query('sm') {
+		display: flex;
+		flex-wrap: wrap;
+		padding: spacing(16);
+	}
+
+	@include media-query('md') {
+		flex-wrap: nowrap;
+	}
+}
+
+.c-facet--small {
+	font-size: 1.4rem;
+}
+
+.c-facet__icon {
+	@include u-icon();
+	flex: 0 0 auto;
+}
+
+.c-facet__label {
+	display: block;
+	font-weight: $context--font-weight-bold;
+	white-space: nowrap;
+	margin-right: spacing(24);
+	margin-bottom: spacing(8);
+}
+
+.c-facet__item {
+	margin-bottom: spacing(8);
+
+	@include media-query('sm') {
+		margin-right: spacing(4);
+
+		&:last-of-type {
+			margin-right: spacing(8);
+		}
+	}
+
+	@include media-query('md') {
+		margin-right: spacing(16);
+		margin-bottom: 0;
+		position: relative;
+	}
+}
+
+.c-facet__item legend {
+	width: 100%;
+
+	@include media-query('sm') {
+		width: auto;
+	}
+}
+
+.c-facet__button {
+	align-items: center;
+	color: black;
+	background: white;
+	border: 1px solid black;
+	border-radius: 3px;
+	display: flex;
+	font-weight: $context--font-weight-normal;
+	margin-bottom: 0;
+	padding: spacing(8);
+	white-space: normal;
+	width: 100%;
+
+	@include media-query('sm') {
+		margin-bottom: 0;
+		white-space: nowrap;
+		width: 111px; // hardcoded to fit 4 items in a row
+	}
+
+	@include media-query('md') {
+		width: 136px; // hardcoded to fit 4 items in a row
+	}
+
+	@include media-query('lg') {
+		width: 190px; // hardcoded to fit 4 items in a row
+	}
+
+	> svg {
+		margin-left: auto;
+		transition-duration: 0.2s;
+	}
+
+	&.is-open {
+		text-decoration: none;
+
+		svg {
+			transform: rotate(180deg);
+		}
+	}
+}
+
+.c-facet__button--disabled {
+	border-color: #c7c4bf;
```

Comment:
Should this colour be variablised because it's possible that other brands will want to overwrite the disabled colour?

Reply:
I'll see if there is an existing colour for this and the comment below in the toolkit otherwise I'll put it a variable

Reply:
This is living as variables in settings. Did see that we have disabled button theme but it is not suitable for the nature brand since it is not on a white background.